### PR TITLE
test, integ: Add dynamic test for IPv4 and IPv6 without server

### PR DESCRIPTION
**Pull request by almusil**
Signed-off-by: Ales Musil <amusil@redhat.com>

### CODE REVIEW

File: tests/integration/dynamic_ip_test.py [hunk 1]
```diff
@@ -102,6 +102,12 @@
     DHCP_SRV_NIC
 )
 
+parametrize_ip_ver = pytest.mark.parametrize(
+    "nm_ip",
```
Comment (EdDev) [inline tests/integration/dynamic_ip_test.py]: I think you meant `ip_ver`.

File: tests/integration/dynamic_ip_test.py [hunk 2]
```diff
@@ -839,12 +845,17 @@ def dummy00():
     libnmstate.apply({Interface.KEY: [ifstate]}, verify_change=False)
 
 
-def test_activate_dummy_without_dhcp_service(dummy00):
+@parametrize_ip_ver
+def test_activate_dummy_without_dhcp_service(ip_ver, dummy00):
+    if ip_ver != (Interface.IPV4, Interface.IPV6):
+        pytest.xfail('https://bugzilla.redhat.com/1798374', strict=True)
```
Comment (EdDev) [inline tests/integration/dynamic_ip_test.py]: You should place the xfails in the `parametrize`.

File: tests/integration/dynamic_ip_test.py (hunk 1 above)
Comment (almusil) [reply tests/integration/dynamic_ip_test.py]: Done, thanks

File: tests/integration/dynamic_ip_test.py (hunk 2 above)
Comment (almusil) [reply tests/integration/dynamic_ip_test.py]: Done, thanks

**Comment by cathay4t** [comment]
Closing as patch merged into PR #749

**Pull request closed without merging**